examples/example.krun

```diff
@@ -75,5 +75,5 @@ TEMP_READ_PAUSE = 1
 #PRE_EXECUTION_CMDS = ["sudo service cron stop"]
 #POST_EXECUTION_CMDS = ["sudo service cron start"]
 
-# CPU pinning (off by default)
-#ENABLE_PINNING = False
+# CPU pinning (on by default)
+#ENABLE_PINNING = True
```

Comment:
At the moment I wouldn't worry too much about this at the moment. In the long run, I tend to prefer man pages, but I don't feel strongly about it.